### DISCUSSION TITLE
用皮膚操作穿戴式裝置

### DISCUSSION THREAD

**Discussion by 11timyyy** [1101-GS4719-C/Python-Community-GS4719-C#488] [General]
GOOGLE於2020年時提出一項專利，其為利用皮膚來增加使用穿戴式裝置時，能夠擴大操作面積，方便操作。
這項專利結合多感測器數據的 Sensor Fusion 技術，提供更準確控制，也利用機器學習減少誤判的發生，如說話吃飯等等。
個人認為這項專利值得期待，因為我也會使用穿戴式裝置，若推行後可以體驗看看!

文章來源:https://technews.tw/2022/03/11/google-pixel-watch-en-pixel-buds-met-skin-interface/

**Comment by ThomasMeng0515** [comment]
之前好像也有人發過類似的文章，不知道如果皮膚流汗會不會影響到感測以至於造成誤判~

**Reply by 11timyyy** [reply]
啊竟然重複了我很抱歉，下次我會多注意😥😥

**Comment by chupochun** [comment]
身體穿戴裝置雖然方便，但是會害怕會受到電磁輻射的影響

**Comment by hapiwangy** [comment]
每個人的身體情況不同，越貼近人皮膚的裝置想要能給所有人使用的挑戰上就越高

**Comment by HuanYuuuuuu** [comment]
不知道長期下來會不會有皮膚過敏或是其他皮膚的症狀

**Comment by PoooYen** [comment]
不知道毛太多會不會影響穿戴

**Comment by sys511613** [comment]
之前sony推出的[WF-L900](https://store.sony.com.tw/product/WF-L900)就有這樣的功能，點耳朵附近就可以進行操作，我覺得超酷
![image](https://user-images.githubusercontent.com/73024991/173362697-f1365b76-8d64-41c6-bbd1-2531901d8681.png)

**Comment by treeann** [comment]
穿戴式裝置一直是我的愛

**Comment by stan890425** [comment]
以後就可以在皮膚上划手機了~~

**Comment by jcpan821** [comment]
搭公車很方便

**Comment by Ben6uo** [comment]
如果再加合個藍牙眼鏡就可以藏起來玩個爽了

**Comment by alexthebigpp** [comment]
穿戴式裝置感覺是目前的趨勢~
